sway/commands.c

```diff
@@ -229,8 +229,8 @@ static bool cmd_set(struct sway_config *config, int argc, char **argv) {
 
 static bool _do_split(struct sway_config *config, int argc, char **argv, int layout) {
 	char *name = layout == L_VERT  ? "splitv":
-	             layout == L_HORIZ ? "splith":
-	                                 "split";
+				 layout == L_HORIZ ? "splith":
+									 "split";
```

Comment:
Push the tabs further to the left and join the two halves of the ternary statement.